### ISSUE TITLE
Dynamic Dialogue mask does not block the underlying UI

### ISSUE THREAD

**Issue by EzaBlade** [primefaces/primeng#10828]
[ x ] bug report => Search github for a similar issue or PR before submitting

When a dialogue is opened using the Dynamic Dialogue service it is still possible to interact with the underlying controls, so I am able to click on buttons etc that should be blocked by the mask.

V12.2.2

**Comment by jayee** [comment]
I can confirm this problem.

**Comment by pkwieci1** [comment]
I confirm, same issue here.

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
